shims.h

```diff
@@ -9,12 +9,6 @@ enum {
     GO_OPENSSL_INIT_LOAD_CONFIG = 0x00000040L
 };
 
-// #include <openssl/aes.h>
-enum {
-    GO_AES_ENCRYPT = 1,
-    GO_AES_DECRYPT = 0
-};
```

Comment:
I think it makes sense to get rid of these particular constants (they aren't accurate given our use and they aren't mentioned in `EVP_CipherInit_ex` docs), but... I kind of wish we still had some constants of our own rather than putting 0 and 1 in the C and Go code. 😅 (And in theory, -1 is also a possibility.)

Reply:
Good point, I'll add that constant. FYI, -1 is documented to be a valid value:

> The operation performed depends on the value of the enc parameter. It should be set to 1 for encryption, 0 for decryption and -1 to leave the value unchanged (the actual value of 'enc' being supplied in a previous call).
https://www.openssl.org/docs/man3.0/man3/EVP_CipherInit_ex.html